search-sentiments-map/routes/trends.js

```diff
@@ -246,8 +246,10 @@ function getGlobalTrends(trendsByCountry) {
   })
 
   let globalTrends = [];
-  // Get the top 10 trends overall.
-  for (let i = 0; i < 10; i++) {
+  let numTopTrends = trendCountsArr.length < 10 ? trendCountsArr.length : 10;
```

Comment:
In reality it's unlikely that there will ever be less than 10 trends in total, but I guess it's needed for the test and there's no harm to check